lib/CMakeLists.txt

```diff
@@ -141,5 +141,11 @@ if(ENABLE_STATIC_LIB)
     DESTINATION "${CMAKE_INSTALL_LIBDIR}")
 endif()
 
+install(
+  EXPORT ${NGTCP2_TARGETS_EXPORT_NAME}
+  NAMESPACE ${NGTCP2_NAMESPACE}
+  DESTINATION ${NGTCP2_CONFIG_INSTALL_DIR}
+)
```

Comment:
Isn't this already present around line ~90 already?

Reply:
No, it different. 

https://stackoverflow.com/questions/71109431/cmake-install-an-export-file-in-debug-release-alternatively
https://stackoverflow.com/questions/45777734/install-export-requires-target-from-subproject

Reply:
Good point.  I missed that.  @talregev could you check that these lines of code?

Reply:
I already test it on vcpkg and it working.

Reply:
@tatsuhiro-t See the answer below.

Reply:
The two snippets _are_ the same (except formatting), so perhaps the location of this instruction is what fixed it for you in vcpkg?

Is such case, what about moving the existing instruction to this new position (keeping formatting), instead of adding a second copy?

Reply:
I can try to move lines 90 to the new location and see if it solve it for vcpkg.

Reply:
Change it.